### DISCUSSION TITLE
Delphi 12 Packages

### DISCUSSION THREAD

**Discussion by PeterPanino** [AngusJohnson/Image32#37] [General]
Does it also work with Delphi 12?

**Comment by AngusJohnson** [comment]
Hi Peter. 
I can't say for sure as I don't have Delphi 12 to test it. But I'd be quite surprised if there were problems loading it in Delphi 12.
There are no specific packages for any Delphi version as it doesn't need to be installed into the IDE. Just follow the installation instructions [here](https://www.angusj.com/image32/Docs/Installation.htm) so the compiler knows where to look for the library.

If, however, you do want to install the separate [TImage32Panel](https://www.angusj.com/image32/Docs/Img32.Panels/_Body.htm) component, then follow the instructions at the bottom of [this](https://www.angusj.com/image32/Docs/Img32.Panels/_Body.htm) page.
